Deep_learning/Deep-learning.rst

```diff
@@ -407,45 +407,44 @@ Note: This will mount your host home directory on /data in the container.
 
 ::
 
-  cd /data/pytorch/
-  python tools/amd_build/build_amd.py
+  python3 tools/amd_build/build_amd.py
 
 6. Build and install pytorch:
 
-Unless you are running a gfx900/Vega10-type GPU (MI25, Vega56, Vega64,...), explicitly export the GPU architecture to build for, e.g.:
-export HCC_AMDGPU_TARGET=gfx906
+By default pytorch is built for all supported AMD GPU targets like gfx803/gfx900/gfx906/gfx908 (MI25, MI50, MI60, MI100, ...)
+This can be overwritten using
+export PYTORCH_ROCM_ARCH=gfx803;gfx900
 
 then
 ::
 
-  USE_ROCM=1 MAX_JOBS=4 python setup.py install --user 
+  USE_ROCM=1 MAX_JOBS=4 python3 setup.py install --user
 
 UseMAX_JOBS=n to limit peak memory usage. If building fails try falling back to fewer jobs. 4 jobs assume available main memory of 16 GB or larger.
 
 7. Confirm working installation:
 
 ::
 
-  PYTORCH_TEST_WITH_ROCM=1 python test/run_test.py --verbose
+  PYTORCH_TEST_WITH_ROCM=1 python3 test/run_test.py --verbose
 
 No tests will fail if the compilation and installation is correct.
 
 8. Install torchvision:
 
 ::
+  pip3 install --user "git+https://github.com/pytorch/vision.git"
 
-  pip install torchvision
-
-This step is optional but most PyTorch scripts will use torchvision to load models. E.g., running the pytorch examples requires torchvision.
+This step is optional. However, most PyTorch scripts will use torchvision to load models. For example, running the PyTorch examples requires torchvision.
 
 9. Commit the container to preserve the pytorch install (from the host):
 
 ::
 
-  sudo docker commit <container_id> -m 'pytorch installed'
+  sudo docker commit <container_id> -m <new image name>
```

Comment:
The `-m` flag is for an optional commit message (https://docs.docker.com/engine/reference/commandline/commit/). The image name at the end is optional too, but if we want to keep that optional argument, it should read: `sudo docker commit <container_id> -m 'pytorch installed' <new image name>`, or  `sudo docker commit <container_id> <new image name>`

Reply:
Its better to revert this line of change will raise it in next PR.

Reply:
https://github.com/RadeonOpenCompute/ROCm_Documentation/pull/125